extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -14,7 +14,8 @@ require([
 	'ext.wikia.adEngine.slotTracker',
 	'ext.wikia.adEngine.slotTweaker',
 	'wikia.krux',
-	'wikia.window'
+	'wikia.window',
+	'ext.wikia.adEngine.adInContentPlayer'
```

Comment:
Actually, if I add `wgEnableIncontentPlayerSlot` I could load it on-demand in Oasis by using our hook. Since `wgIncontentPlayerCountries` is an `InstantGlobal` I don't think it's a good idea to go with a condition:
```php
if( !empty($wgIncontentPlayerCountries) ) {
// add asset group
}
```